src/constants/tokens.ts

```diff
@@ -265,7 +265,7 @@ class MaticNativeCurrency extends NativeCurrency {
 
   get wrapped(): Token {
     if (!isMatic(this.chainId)) throw new Error('Not matic')
-    return WRAPPED_NATIVE_CURRENCY[this.chainId]
+    return WRAPPED_NATIVE_CURRENCY[this.chainId] as Token
```

Comment:
Should you check this? Or use an invariant? Don't cast as it might hide errors from the type system.